cookbook/configuration/pdo_session_storage.rst

```diff
@@ -172,10 +229,10 @@ following (MySQL):
 .. code-block:: sql
 
     CREATE TABLE `session` (
-        `session_id` VARBINARY(128) NOT NULL PRIMARY KEY,
-        `session_data` BLOB NOT NULL,
-        `session_time` INTEGER UNSIGNED NOT NULL,
-        `session_lifetime` MEDIUMINT NOT NULL
+        `sess_id` VARBINARY(128) NOT NULL PRIMARY KEY,
+        `sess_data` BLOB NOT NULL,
```

Comment:
It took me quite some time to find the cause of the auto-logout as there is no error. Good or bad design decision, it still should be noted imo.